jenkins/jobs/bml_integration_tests.pipeline

```diff
@@ -27,7 +27,7 @@ pipeline {
   }
   agent { label 'metal3-static-workers' }
   environment {
-    AIRSHIP_CI_USER="metal3ci"
+    METAL3_CI_USER="metal3ci"
```

Comment:
I am actually surprised that we have this variable here + in line 81. I think this is redundant, but let's not tackle it right now.